README.rst

```diff
@@ -1,4 +1,4 @@
-:lib:`sky130_fd_pr_reram` - SKY130 IO and periphery cells (SkyWater Provided)
+:lib:`sky130_fd_pr_reram` - SKY130 ReRAM and examples (SkyWater Provided)
 =======================================================================
```

Comment:
Header line should be the same length as the title (original file was wrong too it seems)?

Otherwise LGTM.